lib/metababel/bt2_trace_class_generator.rb

```diff
@@ -438,14 +438,40 @@ def get_getter(field:, arg_variables:)
       bt_func_get = self.class.instance_variable_get(:@bt_func) % 'get'
       variable = bt_get_variable(arg_variables).name
       cast_func = @cast_type ? "(#{@cast_type})" : ''
-      pr "#{variable} = #{cast_func}#{bt_func_get}(#{field});"
+
+      if @cast_type and @cast_type.start_with?("struct ")
```

Comment:
Should those function better be in the specialization of `get_getter` / `get_setter` of `BTFieldClass::String` ?
I think it will read better in this case